Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_ResolveAzureFunction.cs

```diff
@@ -84,15 +84,14 @@ private AzureFunctionSpec[] GetAzureFuncDefinitions(IList<FlowGuiFunction> funct
             {
                 var properties = f.Properties;
                 Ensure.NotNull(properties, $"guiConfig.process.functions['{f.Id}'].properties");
-                Ensure.NotNull(properties.Name, $"guiConfig.process.functions['{f.Id}'].properties.name");
                 Ensure.NotNull(properties.ServiceEndpoint, $"guiConfig.process.functions['{f.Id}'].properties.serviceEndpoint");
                 Ensure.NotNull(properties.Api, $"guiConfig.process.functions['{f.Id}'].properties.api");
                 Ensure.NotNull(properties.Code, $"guiConfig.process.functions['{f.Id}'].properties.code");
```

Comment:
Remove the properties.Code check from here as well since we want to support no key scenario

Reply:
This validation is still valid since properties.Code shouldn't be null. For no key, Code is just empty.